fixtures/parallelizer/remote.py

```diff
@@ -233,7 +242,7 @@ def _init_config(slave_options, slave_args):
         conf.runtime["cfme_data"]["basic_info"]["appliance_template"] = template_name
         conf.runtime["cfme_data"]["basic_info"]["appliances_provider"] = provider_name
     config = _init_config(slave_options, slave_args)
-    slave_manager = SlaveManager(config, args.slaveid, appliance.url,
+    slave_manager = SlaveManager(config, args.slaveid, appliance_config,
```

Comment:
**optional** the creation of the appliance beforehand makes me wonder if there is an option of a followup with more lean logic striucture

Reply:
I agree, it all looks a bit funky, but I need to get an understanding of why it was all done this way first :P Added a TODO, can address with another PR to keep the clutter low in this one.

Reply:
thank you 👍 that addresses my thought nicely